src/App.tsx

```diff
@@ -10,33 +12,23 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTab] = useState(tabs[0].id);
+
+  const handleTabSelected = (tab: Tab) => {
+    setSelectedTab(tab.id);
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        {`Selected tab is ${tabs.find(tab => tab.id === selectedTabId)?.title || ''}`}
```

Comment:
just store a whole tab here instead do like this
```suggestion
        {`Selected tab is ${selectedTab.title}`}
```